packaging/ubuntu-16.04/rules

```diff
@@ -139,6 +139,12 @@ ifneq ($(shell dpkg-architecture -qDEB_HOST_ARCH),powerpc)
 	$(shell	if ldd _build/bin/snap-update-ns; then false "need static build"; fi)
 endif
 
+	# ensure snap-seccomp is build with a static libseccomp
+	sed -i "s|#cgo LDFLAGS:|#cgo LDFLAGS: /usr/lib/$(shell dpkg-architecture -qDEB_TARGET_GNU_TYPE)/libseccomp.a|" _build/src/$(DH_GOPKG)/cmd/snap-seccomp/main.go
+	(cd _build/bin && GOPATH=$$(pwd)/.. CGO_LDFLAGS_ALLOW="/.*/libseccomp.a" go build $(GCCGOFLAGS) $(DH_GOPKG)/cmd/snap-seccomp)
```

Comment:
An alternative would be to preserve LDFLAGS in main.go and try:
```
CGO_LDFLAGS_ALLOW='-Wl,-B(static|dynamic)' \
    go build -x -v github.com/snapcore/snapd/cmd/snap-seccomp
```
AFAICT `-Wl,-Bstatic` will trigger use of libseccomp.a even without `pkg-config --static ..`

Reply:
Yeah, this is a good idea as well. We need to modify #cgo LDFLAGS in main.go though or the builds on all the other packages we support will fail with go 1.9 (as well as go get). I picked using the .a file mostly because it seems simpler now that we only do it on ubuntu.